internal/services/codesigning/trusted_signing_certificate_profile_resource_test.go

```diff
@@ -0,0 +1,245 @@
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctest-rg-%[1]d"
+  location = "%[2]s"
+}
+resource "azurerm_trusted_signing_account" "test" {
+  name                = "acctest-%[3]s"
+  location            = "%[2]s"
+  resource_group_name = azurerm_resource_group.test.name
+  sku_name            = "Basic"
+}
+`, data.RandomInteger, data.Locations.Primary, data.RandomString)
+}
+
+func (r TrustedSigningCertificateProfileResource) basic(data acceptance.TestData) string {
+	template := r.template(data)
+	identityId := os.Getenv("ARM_TEST_TRUSTED_SIGNING_IDENTITY_ID")
+	return fmt.Sprintf(`
+				%s
+
+resource "azurerm_trusted_signing_certificate_profile" "test" {
+  name                       = "acctest-ccp-%d"
+  trusted_signing_account_id = azurerm_trusted_signing_account.test.id
+  identity_validation_id     = "%s"
+  profile_type               = "PrivateTrust"
+}
+`, template, data.RandomInteger, identityId)
+}
+
+func (r TrustedSigningCertificateProfileResource) requiresImport(data acceptance.TestData) string {
+	config := r.basic(data)
+	return fmt.Sprintf(`
+			%s
+
+resource "azurerm_trusted_signing_certificate_profile" "import" {
+  name                       = azurerm_trusted_signing_certificate_profile.test.name
+  trusted_signing_account_id = azurerm_trusted_signing_certificate_profile.test.trusted_signing_account_id
+  identity_validation_id     = azurerm_trusted_signing_certificate_profile.test.identity_validation_id
+  profile_type               = azurerm_trusted_signing_certificate_profile.test.profile_type
+}
+`, config)
+}
+
+func (r TrustedSigningCertificateProfileResource) complete(data acceptance.TestData) string {
+	template := r.template(data)
+	identityId := os.Getenv("ARM_TEST_TRUSTED_SIGNING_IDENTITY_ID")
+	return fmt.Sprintf(`
+			%s
+
+resource "azurerm_trusted_signing_certificate_profile" "test" {
+  name                       = "acctest-ccp-%d"
+  trusted_signing_account_id = azurerm_trusted_signing_account.test.id
+  identity_validation_id     = "%s"
+  include_city               = true
+  include_country            = true
+  include_postal_code        = true
+  include_state              = true
+  include_street_address     = true
+  profile_type               = "PrivateTrust"
+
+}
+`, template, data.RandomInteger, identityId)
+}
+
+func (r TrustedSigningCertificateProfileResource) update(data acceptance.TestData) string {
+	template := r.template(data)
+	identityId := os.Getenv("ARM_TEST_TRUSTED_SIGNING_IDENTITY_ID")
+	return fmt.Sprintf(`
+			%s
+
+resource "azurerm_trusted_signing_certificate_profile" "test" {
+  name                       = "acctest-ccp-%d"
+  trusted_signing_account_id = azurerm_trusted_signing_account.test.id
+  identity_validation_id     = "%s"
+  include_city               = false
+  include_country            = false
+  include_postal_code        = false
+  include_state              = false
+  include_street_address     = false
+  profile_type               = "PrivateTrust"
```

Comment:
Can we update this as it is not force new?

Reply:
it was updated, the `complete()` test set all of them to `true`

Reply:
Sorry for the wrong range, i meant the `profile_type` only.